### PR TITLE
on blur fix vote amount automatically

### PR DESCRIPTION
**Pull request by alex-cory**
works, will refactor later w/ mobx

### CODE REVIEW

File: src/scenes/Event/components/EventOption/index.js [hunk 1]
```diff
@@ -51,61 +51,18 @@ export default class EventOption extends Component {
...
-                </Typography>
-              </div>
-            </ExpansionPanelSummary>
-            {showAmountInput && this.renderAmountInput()}
-            {showAmountInput && this.renderAddrSelect()}
-          </ExpansionPanel>
-        </div>
-      </Collapse>
-    );
+  handleAmountBlur = ({ target: { value } }) => {
```
Comment (alex-cory) [inline src/scenes/Event/components/EventOption/index.js]: added this method

File: src/scenes/Event/components/EventOption/index.js [hunk 2]
```diff
@@ -129,6 +86,7 @@ export default class EventOption extends Component {
               className={classes.eventOptionInput}
               onChange={this.handleAmountChange}
               endAdornment={<InputAdornment position="end">{token}</InputAdornment>}
+              onBlur={this.handleAmountBlur}
```
Comment (alex-cory) [inline src/scenes/Event/components/EventOption/index.js]: added this

File: src/scenes/Event/scenes/oracle.js [hunk 3]
```diff
@@ -242,6 +242,7 @@ export default class OraclePage extends Component {
                       isPrevResult={item.isPrevResult}
                       isFinalizing={item.isFinalizing}
                       walletAddresses={this.props.walletAddresses}
+                      consensusThreshold={oracle.consensusThreshold}
```
Comment (alex-cory) [inline src/scenes/Event/scenes/oracle.js]: added this

**Comment by amazingandyyy** [comment]
refer to #720

File: src/scenes/Event/components/EventOption/index.js [hunk 4]
```diff
@@ -51,61 +51,18 @@ export default class EventOption extends Component {
...
-              </div>
-            </ExpansionPanelSummary>
-            {showAmountInput && this.renderAmountInput()}
-            {showAmountInput && this.renderAddrSelect()}
-          </ExpansionPanel>
-        </div>
-      </Collapse>
-    );
+  handleAmountBlur = ({ target: { value } }) => {
+    let { amount, consensusThreshold, onAmountChange } = this.props; // eslint-disable-line
```
Comment (amazingandyyy) [inline src/scenes/Event/components/EventOption/index.js]: what's wrong with eslint here? use `const` instead of `let`?

**Pull request merged**